website/content/en/docs/migrating-existing-apis.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Migrating Existing Kubernetes APIs
-linkTitle: Migrating existing APIs
-weight: 3
+linkTitle: Migrating Existing K8S APIs
+weight: 50
 ---
```

Comment:
The Migrating Existing Kubernetes APIs is valid for all types. So, moving this one for the main menu.